api/scpca_portal/models/sample.py

```diff
@@ -35,8 +35,14 @@ class Modalities:
             SPATIAL: "Spatial Data",
         }
 
+    class AgeTimings(models.TextChoices):
+        COLLECTION = "collection"
+        DIAGNOSIS = "diagnosis"
+        UNKNOWN = "unknown"
```

Comment:
This feels to me like a more concise design decision than taking the other `CHOICES` approach, which would look as follows:
```
class AgeTimings:
    COLLECTION = "collection"
    DIAGNOSIS = "diagnosis"
    UNKNOWN = "unknown"

    CHOICES = (
        (COLLECTION, "Collection"),
        (DIAGNOSIS, "Diagnosis"),
        (UNKNOWN, "Unknown"),
    )
```